runtime/ftplugin/spec.vim

```diff
@@ -113,7 +113,10 @@ if !exists("*s:SpecChangelog")
 			endif
 		endif
 		if (chgline != -1)
+			let tmptime = v:lc_time
+			execute "language time C"
```

Comment:
This line does not need `:execute`. And next new line can be written as

    execute 'language time' tmptime

: -2 spaces and -1 dot.

And in any case you need to contact maintainer listed at the top of the file because Bram accepts changes only from maintainer unless he is unreachable.